orttraining/orttraining/core/dll/orttraining_dll.def

```diff
@@ -0,0 +1,6 @@
+LIBRARY "onnxruntime.dll"
```

Comment:
You can make a small change to https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/gen_def.py , to allow generating this def file from it. Then it will be more portable. The script can support Linux too.

Reply:
I'm not as familiar with the gen_def.py as I'm currently building from the orttraining.win.sln.  If you have a suggested change for the gen_def.py, feel free to make the change needed.  

Also, I currently only have CMakeList.txt files (generated from the *.sln using cmake-converter that build the non CUDA files, and for that reason have not included them yet.  Any suggestions on adding the CUDA file support in would be helpful.

Reply:
It's fine. We can do it later. And I think eventually we will convert all the vs project files to cmake files. Because for us Windows and Linux are equally important.